src/ol/style/stroke.js

```diff
@@ -220,10 +218,6 @@ ol.style.Stroke.prototype.getChecksum = function() {
             this.miterLimit_.toString() : '-') + ',' +
         (this.width_ !== undefined ?
             this.width_.toString() : '-');
-
-    var md5 = new goog.crypt.Md5();
-    md5.update(raw);
-    this.checksum_ = goog.crypt.byteArrayToString(md5.digest());
```

Comment:
Crazy.  Would be interesting to look at the build size before and after this commit alone.